packages/governance/tools/fakeContractGovernor.js

```diff
@@ -0,0 +1,95 @@
+import '../src/types-ambient.js';
+
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+
+// eslint-disable-next-line no-unused-vars
+import { CONTRACT_ELECTORATE } from '../src/contractGovernance/governParam.js';
+
+// @file fakeContractGovernor is a simplified copy of ContractGovernor to
```

Comment:
The naming here is a little murky to me.

While this is simpler than ContractGovernor, it does seem to be a legitimate _contract governor_. It just doesn't do so by electorate, the way `contractGovernor.js` does. 

https://github.com/Agoric/agoric-sdk/blob/91134411c97c4c120c25aa21aa645a986468046d/packages/governance/src/contractGovernor.js#L65-L68

I like what you did above with "could be called fakeCommittee" but I don't object to the naming here. We can refer to `ContractGovernor` as the particular contract in that file, and "fake" is used throughout the codebase to mean an alternative version for testing. Though consider something like this,

```
// @file a version of the contractGovernor.js contract simplified for testing.
// It removes the electorate and doesn't try to support legibility.
// It maintains the API for the governed contract (parameters, apis, and filters)
// It adds the ability for tests to update parameters directly.
```

Reply:
The sense in which it isn't a *legitimate* contractGovernor is that it doesn't support the legibility required in actual governance. I was trying to follow the [standard terminology for testing utilities](https://stackoverflow.com/questions/346372/whats-the-difference-between-faking-mocking-and-stubbing), which talks about mocks, stubs, and fakes. It's clearly not a mock or a stub. Fakes can have an actual implementation, which is simplified from the real thing, though Fakes often regurgitate canned data. Your suggestion of "Puppet" seems to clarify that it's controlled by the test.